commit 930a321f8e411cd02f4f43c193f967763e8b6e5c
Author: Cole Robinson <crobinso@redhat.com>
Date:   Fri Oct 28 12:20:11 2011 -0400

    console: Fix crashes when deleting vm on F16
    
    The toolbar cleanup order was causing crashed on f16 at app shutdown or
    VM delete (when a console window is fully uninstantiated)

diff --git a/src/virtManager/console.py b/src/virtManager/console.py
index d6d95cd..559bb31 100644
--- a/src/virtManager/console.py
+++ b/src/virtManager/console.py
@@ -603,10 +603,10 @@ class vmmConsolePages(vmmGObjectUI):
 
         self.keycombo_menu.destroy()
         self.keycombo_menu = None
-        self.fs_toolbar.destroy()
-        self.fs_toolbar = None
         self.fs_drawer.destroy()
         self.fs_drawer = None
+        self.fs_toolbar.destroy()
+        self.fs_toolbar = None
 
     ##########################
     # Initialization helpers #
